gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c

```diff
@@ -171,75 +173,101 @@ custom_open (const GstTensorFilterProperties * prop, void **private_data)
 /**
  * @brief Callback required by tensor_filter subplugin
  */
-static int
-custom_invoke (const GstTensorFilterProperties * prop,
-    void **private_data, const GstTensorMemory * input,
-    GstTensorMemory * output)
+static void
```

Comment:
The return type is 'void', did you intend?

Reply:
It's defined in `nnstreamer_plugin_api_filter.h`
https://github.com/nnstreamer/nnstreamer/blob/e2494ce89baf53ef8ce80e5272c71512891c89fe/gst/nnstreamer/include/nnstreamer_plugin_api_filter.h#L291